kentikapi/credentials.go

```diff
@@ -4,31 +4,55 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
 	"path/filepath"
 )
 
+type Profile struct {
```

Comment:
This type is an implementation detail, not meant to be consumed by users so please unexport it (i.e. make private) and move next to readCredentialsFromProfile() function. I would put it below readCredentialsFromProfile function.

Reply:
Done